CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 ### Bug Fixes
 
-_None_
+* Fixed `swiftlint` warnings in generated color extensions
```

Comment:
Please use the same format as the other entries:
- credit yourself
- add a link to this PR
- a `.` at the end of the sentence and 2 spaces